### PR TITLE
[WIP] Hardening Guide: update and simplify intro

### PR DESCRIPTION
**Pull request by taroth21**
### Description
As of bsc#1152504, updating the introduction to the Common Criteria chapter (1.1): removing details about old product versions and simplifying sentences.    

### Checklist
* Check all items that apply.

*Are backports required?*

- [x] To maintenance/SLE15SP1
- [x] To maintenance/SLE15SP0
- [x] To maintenance/SLE12SP5
- [x] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3

### CODE REVIEW

**Comment by taroth21** [comment]
Draft PDF of the updated chapter: [cha-common-criteria_color_draft_en.pdf](https://github.com/SUSE/doc-sle/files/3813587/cha-common-criteria_color_draft_en.pdf)

**Comment by taroth21** [comment]
Draft PDF of the latest changes, including two questions to the subject matter experts: 
[cha-common-criteria_remarks_color_draft_en.pdf](https://github.com/SUSE/doc-sle/files/3824941/cha-common-criteria_remarks_color_draft_en.pdf)

File: xml/hardening.xml [hunk 1]
```diff
@@ -340,31 +340,30 @@
...
 
    <para>
     A clear definition of security in IT products is challenging. Security
     should be considered a process that never ends, not a static condition
-    that can be met or not. A Common Criteria certificate (below EA7) does not
-    make a clear statement about error proneness of the system (while many of
-    the flaws that exist specifically in operating systems are
-    security-relevant), but it adds an important value to the product that
+    that can be met or not. A Common Criteria certificate (below EA7)
+    <remark>taroth 2019-11-07: shouldn't that read *EAL7*?</remark>does not
```
Comment (cjschroder) [inline xml/hardening.xml]: Yes, it is EAL7

File: xml/hardening.xml [hunk 2]
```diff
@@ -590,14 +496,29 @@
...
       <para>
-       Generally, an RPM software package consists of the package's meta data
-       that is written to the RPM database upon installation, the package's
-       files and directories and scripts that are being executed before and
-       after installation and removal.
+       Generally, an RPM software package consists of the following:
       </para>
+      <itemizedlist>
+       <listitem>
+        <para>The package's meta data that is written to the RPM database upon
```
Comment (cjschroder) [inline xml/hardening.xml]: "meta data" should be "metadata"

File: xml/hardening.xml [hunk 3]
```diff
@@ -649,135 +566,146 @@
...
+       take much effort to configure them.
       </para>
       <para>
-       In particular, the use of
-       <systemitem class="resource">AIDE</systemitem> is strongly encouraged.
-       <systemitem class="resource">AIDE</systemitem>, when run for
-       initialization, creates a hash database of all files in the system that
-       are listed in its configuration file. This allows to verify the
-       integrity of all cataloged files at a later time.
+       In particular, we recommend to use the file and directory integrity checker
```
Comment (cjschroder) [inline xml/hardening.xml]: "to use" should be "using"

File: xml/hardening.xml [hunk 4]
```diff
@@ -649,135 +566,146 @@
...
+       <systemitem class="resource">AIDE</systemitem> (Advanced Intrusion Detection Environment).
+       When run for initialization, it creates a hash database of all files in the system that
+       are listed in its configuration file. This allows to verify the integrity
```
Comment (cjschroder) [inline xml/hardening.xml]: "to verify" should be "verifying"

File: xml/hardening.xml [hunk 5]
```diff
@@ -649,135 +566,146 @@
...
-        entire variety of consequences: Apart from being very hard to detect,
+        An attacker may also have planted a backdoor in the kernel. Apart from
+        being very hard to detect,
         the kernel-based backdoor can effectively remove all traces of the
-        system compromise to the degree that system alterations are almost
-        invisible. By consequence, an integrity check needs to be done from a
-        rescue system (or any other system where an independent system runs,
-        and the target system's file systems are mounted manually).
+        system compromise so system alterations become almost invisible.
+        By consequence, an integrity check needs to be done from a
```
Comment (cjschroder) [inline xml/hardening.xml]: "By consequence" should be "Consequently"

File: xml/hardening.xml (hunk 1 above)
Comment (taroth21) [reply xml/hardening.xml]: Yep, also confirmed by Ivan in Bugzilla, fixed.

File: xml/hardening.xml (hunk 2 above)
Comment (taroth21) [reply xml/hardening.xml]: Hey, you have read the style guide! :) Fixed.

**Comment by taroth21** [comment]
Backported to the following versions:

SLE 15 SP1:
https://github.com/SUSE/doc-sle/commit/4c65173de412165d72e9c6ff69a3575b9370b6eb

SLE 15:
https://github.com/SUSE/doc-sle/commit/82c62ce579b14617d1a09c3644aeea346a54fe93

SLE 12 SP5:
https://github.com/SUSE/doc-sle/commit/f26524eec61b0bdbabb768ff18f9e12644917052

SLE 12 SP4:
https://github.com/SUSE/doc-sle/commit/e9b45c6506f3be2f14c68bf31d72ac0b7f10e9b6

**Pull request merged**